documentation/adr/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,19 @@
+# 1. Record architecture decisions
```

Comment:
Should these be in https://github.com/DFE-Digital/cpd-adr?

Reply:
I don't think we're going to be using that repo.... @mattonfoot ?